### ISSUE TITLE
Put Moment.js as an optional dependency

### ISSUE THREAD

**Issue by josemanuelp** [quasarframework/quasar#311]
Hi.

Congratulations on this great work.

I wonder if it's possible to put moment.js as an optional dependency when not using datetime components.

Moment.js is huge!
```
Index of /min/

Name	Type	Size	Last Modified
..	-	-	-
locales.js	application/javascript	303 kB	2016-12-04T05:48:31.000Z
locales.min.js	application/javascript	194 kB	2016-12-04T05:48:31.000Z
moment-with-locales.js	application/javascript	426 kB	2016-12-04T05:48:31.000Z
moment-with-locales.min.js	application/javascript	255 kB	2016-12-04T05:48:31.000Z
moment.min.js	application/javascript	61.3 kB	2016-12-04T05:48:31.000Z
```

Or at least find a way to select certain locations. I think in this way the footprint of QuasarFramework is going to be much less.

Thanks.

**Comment by rstoenescu** [comment]
In `/build/weback.base.conf.js` there's a section specifically for this:

```js
/* Uncomment if you wish to load only one Moment locale: */
// new webpack.ContextReplacementPlugin(/moment[\/\\]locale$/, /en/),
```

Will do something about this, yeah. This is part of the roadmap (import only needed components). Thanks!

**Comment by smolinari** [comment]
How hard would it be to make the single locale selection dynamic?

Scott

**Comment by rstoenescu** [comment]
@smolinari Due to how Momentjs is built: extremely hacky

**Comment by smolinari** [comment]
Hmm.....interesting.

Scott

**Comment by rstoenescu** [comment]
This has been moved to v0.14

**Comment by rstoenescu** [comment]
Changes will be available in future v0.14.
